encoding/fft/fft.go

```diff
@@ -88,3 +90,9 @@ func NewFFTSettings(maxScale uint8) *FFTSettings {
 		ReverseRootsOfUnity:  rootzReverse,
 	}
 }
+
+// FFTSettingsFromBlobLength accepts a blob length in symbols, and returns a new instance of FFT settings
+func FFTSettingsFromBlobLength(blobLength uint32) *FFTSettings {
```

Comment:
would really like blobLength to have a suffix, I feel it'll come back to byte (pun unintended at first lol) us otherwise

Reply:
Agreed, will change pending resolution of discussion [above](https://github.com/Layr-Labs/eigenda/pull/1120#discussion_r1954877635)